weave/wandb_file_manager.py

```diff
@@ -57,15 +66,15 @@ def _local_path_and_download_url(
     file_path = _file_path(art_uri, md5_hex)
     if manifest.storage_layout == artifact_wandb.WandbArtifactManifest.StorageLayout.V1:
         return file_path, "{}/artifacts/{}/{}/{}".format(
-            base_url, art_uri.entity_name, md5_hex, urllib.parse.quote(file_name)
+            base_url, art_uri.entity_name or "_", md5_hex, urllib.parse.quote(file_name)
```

Comment:
is this `_` valid? Or just a way to fix typing?

Reply:
`_` is valid now. The backend artifact file download handler actually ignores the entity name param altogether, so i can actually pass in any arbitrary string, as long as its not None or an empty string.

https://github.com/wandb/core/blob/7cfee1cd07ddc49fe7ba70ce3d213d2a11bd4456/services/gorilla/api/handler/artifacts.go#L179

Reply:
👍 ... can you please add this as a comment for future devs